gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -104,7 +103,7 @@
   @Nullable private final Duration keepAliveTime;
   @Nullable private final Duration keepAliveTimeout;
   @Nullable private final Boolean keepAliveWithoutCalls;
-  @Nullable private final Integer poolSize;
+  private final ChannelPoolSettings channelPoolSettings;
```

Comment:
While we're at it, I see `static final int MAX_POOL_SIZE = 1000` on line 91 is unused. Let's remove it.

Reply:
done